app/models/delete_service_event.rb

```diff
@@ -0,0 +1,18 @@
+class DeleteServiceEvent < AggregatedEvent
+  belongs_to_aggregate :service
+
+  data_attributes :entity_id, :name
+  validates_presence_of :entity_id, :name
```

Comment:
```suggestion
```
we're only passing an existing service object to the event, no attributes or anything, so no need for it